pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -178,6 +178,15 @@ public static class ControllerPeriodicTasksConf {
     public static final String DEPRECATED_SEGMENT_RELOCATOR_FREQUENCY_IN_SECONDS =
         "controller.segment.relocator.frequencyInSeconds";
     public static final String SEGMENT_RELOCATOR_FREQUENCY_PERIOD = "controller.segment.relocator.frequencyPeriod";
+
+    //whether segment relocator should do a best-efforts rebalance. Default is 'true'
+    public static final String SEGMENT_RELOCATOR_BEST_EFFORTS = "controller.segment.relocator.bestEfforts";
+    //For no-downtime rebalance, minimum number of replicas to keep alive during rebalance, or maximum number of
+    // replicas allowed to be unavailable if value is negative. Default value is -1
+    public static final String SEGMENT_RELOCATOR_MIN_AVAIL_REPLICAS = "controller.segment.relocator.minAvailReplicas";
```

Comment:
Suggest keeping the name align with `RebalanceConfig`: `controller.segment.relocator.minAvailableReplicas` and `controller.segment.relocator.bootstrap`